bundle/src/types.rs

```diff
@@ -31,6 +31,7 @@ pub struct Test {
     pub class_name: Option<String>,
     pub file: Option<String>,
     pub id: String,
+    pub timestamp_millis: Option<i64>,
```

Comment:
I'm not sure of the full implications of expanding this type, since there are bindings for it. I can remove it from here if desired, and just store pairs/"max timestamp" in the extraction map